optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -974,6 +974,49 @@
       </section>
     </section>
 
+    <section xml:id="statisticalBenchmarking">
+      <title>Statistical Benchmarking</title>
+
+      <para>If you want to minimize the influence of your environment on the benchmark results, you
+        can configure the number of times each single benchmark run is repeated. The results of
+        those runs will get statistically aggregated and are even visible in the report
+        individually.</para>
+
+      <figure>
+        <title>Sub Single Benchmark Summary Statistic</title>
+
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="images/Chapter-Benchmarking_and_tweaking/subSingleBenchmarkSummaryStatistic.png"/>
+          </imageobject>
+        </mediaobject>
+      </figure>
+
+      <para>To configure this in your benchmarks, add a <literal>&lt;subSingleCount&gt;</literal>
+        element to a <literal>&lt;solverBenchmark&gt;</literal> element:</para>
+
+      <programlisting language="xml">&lt;plannerBenchmark&gt;
+  ...
+  &lt;solverBenchmark&gt;
+    &lt;name&gt;...&lt;/name&gt;
+    &lt;solver&gt;
+      ...
+    &lt;/solver&gt;
+    &lt;subSingleCount&gt;10&lt;subSingleCount&gt;
+  &lt;/solverBenchmark&gt;
+  ...
+&lt;/plannerBenchmark&gt;</programlisting>
```

Comment:
I would show an example with the inheritedSolverBenchmark instead of the solverBenchmark because 99% of the time people will only use it there. So we should explain the most common use case first. See also writing of problemBenchmarks: even though that works on a solverBenchmark, the docs first teach you that it works on inheritedSolverBenchmark (as that's where you'll use it) and only then they briefly mention IIRC that it also works on a solverBenchmark level (without an example as that's just wasting paper space).